telemetry/telemetryformat.md

```diff
@@ -26,9 +26,20 @@ _types_ is an array that holds telemetry metadata types. This is the information
 
 ### Metrics
 
-_metrics_ is an array that contains the actual metrics posted to the service. `name` defines the metric name what is
-posted to the service, `metadata` contains data from `types` that define characteristics of the telemetry beyond
-`createTime` and a `value`. `name` must be in the format`namespace_camelCaseName` (e.g. `s3_uploadObject`). The field is optional.
+_metrics_ is an array that contains the actual metrics posted to the service.
+- `name` defines the metric name
+  - it must be in the format `namespace_camelCaseName` (e.g. `s3_objectUpload`).
+  - it must be in the format `{namespace}_{noun}{Verb}` (eg: `toolkit_moduleInit`)
+    - This format enables a natural grouping of related metrics in our definitions file, as well as improved searchability.
+  - Common Namespaces:
+    - `toolkit_` is for general non-feature-specific metrics created by the toolkit. Eg: `toolkit_moduleOpen`
+    - `ide_` is for IDE specific metrics, not controlled by our extension. Eg: `ide_editorOpen`
```

Comment:
can we also add an entry for ui_click?

Reply:
what do you mean specifically? 

IIUC we semi-agreed in a previous slack thread that with this new spec, `toolkit_uiClick` would technically be the name that we go with **if it didn't already exist**